packages/aragon-wrapper/src/index.js

```diff
@@ -873,6 +873,31 @@ export default class Aragon {
       } catch (_) { }
     }
 
+    return this.calculateDirectPath (sender, destination, directTransaction, finalForwarder)
+
+  }
+
+  /**
+   * Calculate the transaction path for a transaction to `destination`
+   * that invokes `directTransaction`.
+   *
+   * @param  {string} sender
+   * @param  {string} destination
+   * @param  {Object} directTransaction
+   * @param  {string} [finalForwarder]
+   * @return {Array<Object>} An array of Ethereum transactions that describe each step in the path
+   */
+  async calculateDirectPath (sender, destination, directTransaction, finalForwarder) {
```

Comment:
The naming of the function is confusing to me, the intent only got clear after our offline conversation. I think `calculateForwardingPath` would be a way more clear name for describing what this function does.